setup.cfg

```diff
@@ -34,134 +36,142 @@ setup_requires =
 # Dependencies of the project:
 install_requires =
   absl-py==0.14.0
-  # ctcdecode>=1.0.3
-  levenshtein>=0.12.0
-  librosa>=0.8.1
   numpy>=1.19.2
   pandas>=1.3.1
   six>=1.15.0
 python_requires = >=3.7
 
+
+###############################################################################
+#                           Additional Dependencies                           #
+###############################################################################
+
 [options.extras_require]
-# Add extra dependencies, e.g. to run tests or for the different frameworks.
+# Add extra dependencies for individual workloads
+# or developement (linting or testing)
 # Use as `pip install -e '.[jax-gpu]' -f https://storage.googleapis.com/jax-releases/jax_releases.html`
 # or `pip install -e '.[dev]'`
 
+# Bundled installs #
+
+# All workloads
+full =
+  %(librispeech)s
+  %(wmt)s
+
+# All workloads plus development dependencies
+full_dev =
+  %(full)s
+  %(dev)s
+
 # Dependencies for developing the package
 dev =
   isort
   pylint
   pytest
   yapf
 
+
+# Workloads #
+
+librispeech = 
+  ctcdecode>=1.0.2
+  levenshtein>=0.12.0
+  librosa>=0.8.1
+
+wmt =
+  sentencepiece
+
+# Frameworks #
+
 # JAX Core
 jax_core_deps =
   flax==0.3.5
-  jax==0.2.17
   optax==0.0.9
   tensorflow-cpu==2.5.0
   tensorflow_datasets==4.4.0
 
 # JAX CPU
 jax_cpu =
   %(jax_core_deps)s
-  jaxlib==0.1.71
+  jax==0.2.28
+  jaxlib==0.1.76
 
 # JAX GPU
 jax_gpu =
   %(jax_core_deps)s
-  jaxlib==0.1.71+cuda111
+  jax[cuda]==0.2.28
+  jaxlib==0.1.76+cuda11.cudnn82
 
 # PyTorch
 pytorch =
   torch==1.9.1+cu111
```

Comment:
Is there a reason why we don't use the latest stable release (1.10.2)?

Reply:
I don't think there is one. Perhaps this was the stable release at the time of initializing the package.

I also noticed that the learning rate schedule in `imagenet_pytorch` requires `torch>=1.10`. We will probably have to discuss and fix the package versions close to the Call for Submissions.

I am happy to bump the version for now. @runame this would mean `torch==1.10.2+cu111`?

Reply:
Ok, the current version is `torch==1.10.2+cu113` and `torchvision==0.11.3+cu113`. Hopefully this doesn't break anything, but most of the changes should be backwards compatible.